### DISCUSSION TITLE
How to use snippets.conf?

### DISCUSSION THREAD

**Discussion by Pance21** [geany/geany#3258] [General]
I am using Geany 1.38, with Linux Mint 21.

I see I have a file called "snippets.conf" - how does it work? I've tried typing the snippet inside Geany, but nothing happens.

Thanks

**Comment by elextr** [comment]
See the [manual](https://www.geany.org/manual/current/index.html#user-definable-snippets)

**Comment by zhaolinlau** [comment]
> I am using Geany 1.38, with Linux Mint 21.
> 
> I see I have a file called "snippets.conf" - how does it work? I've tried typing the snippet inside Geany, but nothing happens.
> 
> Thanks

I made an instruction on how to setup custom snippets and how to use it, you may refer to this [Geany WebDev Snippets](https://github.com/zhaolinlau/Geany-WebDev-Snippets).

**Comment by elextr** [comment]
Did you type tab? https://www.geany.org/manual/current/index.html#user-definable-snippets
